tests/integration/spark/test_reducer_merge.py

```diff
@@ -0,0 +1,164 @@
+import unittest, PyRDF, ROOT
+
+class ReducerMergeTest(unittest.TestCase):
+    """
+    Integration tests to check the working of merge operations
+    in the reducer function.
+
+    """
+    def setUp(self):
+        """
+        Set up method to select Spark backend
+        for every test.
+
+        """
+        PyRDF.use("spark")
+
+    def test_histo1d_merge(self):
+        """
+        Integration test to check the working of
+        Histo1D merge operation in the reducer.
+
+        """
```

Comment:
There is quite some repetition in these tests. Can we refactor the common part into helper functions and just leave the code that is specific to every test (basically, the creation of the histos/profiles).